src/main/java/com/crowdin/client/bundles/model/BundleAttributes.java

```diff
@@ -0,0 +1,8 @@
+package com.crowdin.client.bundles.model;
+
+import lombok.Data;
+
+@Data
+public class BundleAttributes {
```

Comment:
There is a mistake in the API documentation that will be fixed soon

The bundle attributes should contain the `bundleId` field (integer)

```json
"attributes": {
    "bundleId": 2
},
```